applications/asset_tracker/Kconfig

```diff
@@ -286,6 +286,18 @@ config CLOUD_CONNECT_ERR_REBOOT_S
 
 endmenu # Cloud
 
+menu "nRF Connect for Cloud"
+
+config NRFCLOUD_CLIENT_ID_PREFIX
+	string "Prefix used when constructing the MQTT client ID from the IMEI"
+	default "nrf-"
+
+config NRFCLOUD_CLIENT_ID_PREFIX_LEN
+	int "Length of client ID prefix"
+	default 4
```

Comment:
Is it really needed? `sizeof(CONFIG_NRFCLOUD_CLIENT_ID_PREFIX);` could be used to get the actual prefix length.

Reply:
Some other libs to it like that (I don't know which is better): https://github.com/nrfconnect/sdk-nrf/blob/b1637bd8dcfadb296aa2ce8411c2f8eb85bad5bc/subsys/net/lib/aws_iot/Kconfig#L49

I also like the `sizeof` approach, I think the idea because this does not change at runtime, it's overhead.

Reply:
`sizeof` for me is just a neater solution, first, we don't multiply configs, second, it's safer as there's no risk that the user changes the prefix string w/o modifying the prefix len (which would likely result in a runtime crash).